modules/gapi/samples/text_detection.cpp

```diff
@@ -0,0 +1,705 @@
+#include <algorithm>
+#include <cctype>
+#include <cmath>
+#include <iostream>
+#include <limits>
+#include <numeric>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+#include <opencv2/gapi.hpp>
+#include <opencv2/gapi/core.hpp>
+#include <opencv2/gapi/cpu/gcpukernel.hpp>
+#include <opencv2/gapi/infer.hpp>
+#include <opencv2/gapi/infer/ie.hpp>
+#include <opencv2/gapi/streaming/cap.hpp>
+
+#include <opencv2/highgui.hpp> // CommandLineParser
```

Comment:
?
`CommandLineParser` is core/utility.hpp

Reply:
Fixed, thanks